dangermattic.gemspec

```diff
@@ -53,4 +54,5 @@ Gem::Specification.new do |spec|
   #
   # This will stop test execution and let you inspect the results
   spec.add_development_dependency 'pry'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
I thought we removed that one from release-toolkit (which you presumably copied that from?), because that would prevent us from using CI to release the gem to rubygems on new tags automatically?

Reply:
See also https://github.com/wordpress-mobile/release-toolkit/pull/475

Reply:
Indeed, it is reported by RuboCop and auto-correctable:
```
dangermattic.gemspec:9:1: W: [Correctable] Gemspec/RequireMFA: metadata['rubygems_mfa_required'] must be set to 'true'.
```

Reply:
We just have to disable the `Gemspec/RequireMFA` rule in `.rubocop.yml` then.

Reply:
You can copy/paste the comment from https://github.com/wordpress-mobile/release-toolkit/pull/464/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR61-R65 if you need inspiration 😉